tests/test_interpolation.py

```diff
@@ -343,14 +336,25 @@ def test_env_is_not_cached(monkeypatch: Any) -> None:
         ("{a: 0, b: 1}", {"a": 0, "b": 1}),
         ("[\t1, 2, 3\t]", [1, 2, 3]),
         ("{   a: b\t  }", {"a": "b"}),
-        # resolvers
+        # interpolations
+        ("${parent.sibling}", 1),
+        ("${.sibling}", 1),
+        ("${..parent.sibling}", 1),
+        ("${uncle}", 2),
+        ("${..uncle}", 2),
         ("${oc.env:MYKEY}", 456),
     ],
 )
 def test_decode(monkeypatch: Any, value: Optional[str], expected: Any) -> None:
     monkeypatch.setenv("MYKEY", "456")
-    c = OmegaConf.create({"x": f"${{oc.decode:'{value}'}}"})
-    assert c.x == expected
+    c = OmegaConf.create(
+        {
+            # The node of interest is "node" (others are used to test interpolations).
+            "parent": {"node": f"${{oc.decode:'{value}'}}", "sibling": 1},
```

Comment:
add a trailing comma after 1 multiline formatting (will be easier to read).

Reply:
Done in 9090368